mica/report/report.py

```diff
@@ -472,8 +472,7 @@ def main(obsid):
         os.makedirs(outdir)
 
     jinja_env = jinja2.Environment(
-        loader=jinja2.FileSystemLoader(
-            os.path.join(os.environ['SKA'], 'data', 'mica', 'templates')))
+        loader=jinja2.PackageLoader('mica.report', 'templates'))
```

Comment:
This doesn't seem to work in the presence of the `mica...egg-info` directory in site-packages.  Not sure if I should dig into the PackageLoader or just switch this to a FileSystemLoader that uses a relative path.